sru-templates/manual/azure-sru

```diff
@@ -1,25 +1,26 @@
 #!/bin/sh
-set -x
+set -ex
 # Manually deploy on Azure using Azure CLI client.
 # Validate upgrade to proposed version of cloud-init
 # Check Tracebacks or error conditions on clean boot
- 
-# To be adapted to the SRU to test
-SRU_SERIES="%SRU_SERIES%"
 
-# local values
-# find region with az account list-locations -o table
-REGION="<specify your region name>"
-VNET_NAME="<specify your virtual network name>"
-NETWORK_SECURITY_GROUP="<specify your network security group>"
-NIC_NAME="<specify a preferred nic name>"
-RESOURCE_GROUP="<specify your resource group>"
-BOOT_DIAG="<specify your boot diagnostics group>"
-SSH_KEY="$HOME/.ssh/id_rsa.pub"
+SRU_VARS=~/sru-vars.rc
 
+if [ ! -f $SRU_VARS_FILE ]; then
+   echo "Missing config environment file $SRU_VARS."
+   exit 1
+fi
 
-ADVANCED_NIC_TESTS=true
+. $SRU_VARS
 
+parse_args $0 $@
+assert_expected_vars AZURE_REGION AZURE_VNET_NAME \
+  AZURE_NETWORK_SECURITY_GROUP AZURE_NIC_NAME AZURE_RESOURCE_GROUP \
+  AZURE_BOOT_DIAG SSH_KEY SAMPLE_CLOUDCONFIG
+create_samplecloudconfig
+create_setup_proposed_scripts
+
+ADVANCED_NIC_TESTS=true
```

Comment:
When would this be set to something other than `true`? Should this be a configurable `AZURE_*` variable instead?

Reply:
@OddBloke this would only be set to something other than true during test development, in order to cut down on testing costs while writing/extending the manual test